meson.build

```diff
@@ -691,6 +691,13 @@ if host_machine.system() in ['windows', 'cygwin']
     summary('Windows Multimedia support', winmm_dep.found())
 endif
 
+# X11 Input Extension dependency for ManyMouse library
+xi_dep = optional_dep
+using_x11 = host_machine.system() in ['linux', 'freebsd', 'netbsd', 'openbsd', 'dragonfly']
```

Comment:
Might be able to use your nice OS family simplifier?

```python
using_x11 = os_family_name in ['LINUX', 'BSD']
```

Reply:
Good idea, cleaned up - thanks! :)